rust-src/concordium_base/src/ffi_helpers/ffi_macros.rs

```diff
@@ -11,6 +11,7 @@ macro_rules! macro_derive_binary {
         extern "C" fn $function_name(one_ptr: *$mod $type, two_ptr: *$mod $type) -> u8 {
             let one = from_ptr!(one_ptr);
             let two = from_ptr!(two_ptr);
+            #[allow(clippy::not_unsafe_ptr_arg_deref, clippy::redundant_closure_call)]
```

Comment:
```suggestion
            #[allow(clippy::redundant_closure_call)]
```
the not_unsafe thing is already allowed for the entire function.